automerge-backend/src/change.rs

```diff
@@ -508,12 +508,16 @@ fn decode_change(bytes: Vec<u8>) -> Result<Change, AutomergeError> {
     })
 }
 
-fn decompress_chunk(preamble: &[u8], buf: &[u8]) -> Result<ChangeBytes, AutomergeError> {
-    let mut decoder = DeflateDecoder::new(buf);
+fn decompress_chunk(
+    preamble: Range<usize>,
+    body: Range<usize>,
+    buf: &[u8],
```

Comment:
Could `buf` instead be a `Vec<u8>` since we convert it with `.into()` at the end of this function?